style.css

```diff
@@ -0,0 +1,140 @@
+@import url('http://fonts.cdnfonts.com/css/crete-round');
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+body {
+  font-family: 'Roboto', sans-serif;
+}
+
+.toolbar {
+  display: flex;
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  background: #3c3a39;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.toolbar a {
+  width: 77px;
```

Comment:
:red_circle: Your text logo should appear in a single line. I think you can achieve that by removing the `width` property  or set its value to `auto`:hugs: 

Your Design 
![menu28](https://user-images.githubusercontent.com/26736582/198686893-8a79cbb5-5332-47f6-a66f-f8d0c28fa385.png)

Figma Design
![menufigma28](https://user-images.githubusercontent.com/26736582/198686944-e11d3012-4c8d-4195-8cd0-21f5b5522081.png)

Reply:
Thanks. Fixed.